test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1896,19 +1896,24 @@ class test_ucp_am_nbx_rndv_memtype : public test_ucp_am_nbx_rndv {
         return test_ucp_am_nbx_eager_memtype::get_test_variants(variants);
     }
 
-    void init()
+    void init() override
     {
         modify_config("RNDV_THRESH", "128");
```

Comment:
Can this threshold setting be removed since now `UCP_AM_SEND_FLAG_RNDV` is passed?